src/utils/github.ts

```diff
@@ -94,15 +94,21 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | null> {
     logger.info({ path }, 'GitHub API: Get Content');
-    const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return Buffer.from(response.data.content, 'base64').toString();
-    } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
+    try {
+      const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
+      if (this.isOk(response.status)) {
```

Comment:
There's no `else` clause here, so failure responses are not handled or reported. Is this intentional?

Reply:
The response status always return 200 unless catch an error from `this.octokit.rest.repos.getContent`, we might get 404 status from the error. This octokit Api sucks.

Reply:
In that case the `if` statement can just be removed, because the code is unreachable if it's not OK.